src/AmazonPHP/SellingPartner/Api/ApprovalsApi/OrdersSDKInterface.php

```diff
@@ -1,54 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
huh what happen here? Amazon removed that API?

Reply:
It looks like it yes

Reply:
was it announced anywhere? Is there any recommended upgrade plan? Its a breaking change for the lib and it might require another major release then.